docs/release-notes/NuGet-5.5.md

```diff
@@ -0,0 +1,118 @@
+
+* Lock file is not honored in "*" scenarios  - [#8073](https://github.com/NuGet/Home/issues/8073)
+
+* NuGet.exe does not resolve to the latest version of a package when using * in PackageReference (MSBuild/Dotnet/VS restore do) - [#8432](https://github.com/NuGet/Home/issues/8432)
+
+* dotnet list package with multi targeting WPF project - [#8463](https://github.com/NuGet/Home/issues/8463)
+
+* Plugin:  "A task was cancelled" - problem with ADO authentication due to this. - [#8528](https://github.com/NuGet/Home/issues/8528)
+
+* Improve ConcurrencyUtilities (reduce CPU usage) - [#8653](https://github.com/NuGet/Home/issues/8653)
+
+* DG Spec for unloaded project scenarios should not be written in preview restores - [#8793](https://github.com/NuGet/Home/issues/8793)
+
+* The Visual Studio NuGet packages (RestoreManagerPackage) needs to auto load on solution build events - [#8796](https://github.com/NuGet/Home/issues/8796)
+
+* [Test Failure] Package icon becomes a default icon after installing ‘Newtonsoft.Json’ with latest versoin - [#8814](https://github.com/NuGet/Home/issues/8814)
+
+* Deadlock in VSSettings init - [#8842](https://github.com/NuGet/Home/issues/8842)
+
+* VisualStudio ToolBox is not populated from a NuGet package if a project is placed in a solution folder - [#8868](https://github.com/NuGet/Home/issues/8868)
+
+* VS:  solution restore perpetually fails due to race condition - [#8881](https://github.com/NuGet/Home/issues/8881)
+
+* Constant "loading.." on installed tab, and "searching <term>.." on updates tab - [#8890](https://github.com/NuGet/Home/issues/8890)
+
+* NuGet client tools strategy with multiple feeds on packages with version floats - [#8992](https://github.com/NuGet/Home/issues/8992)
+
+* Accessibility Fixes in PM UI - altText, etc... - [#9059](https://github.com/NuGet/Home/issues/9059)
+
+* Missing Embedded Icons in VS PM UI after cache expires - [#9069](https://github.com/NuGet/Home/issues/9069)
+
+* Icon missing after package install in VS PM UI - [#9072](https://github.com/NuGet/Home/issues/9072)
+
+* Review Accessibility bugs - [#9077](https://github.com/NuGet/Home/issues/9077)
+
+* PMUI Tabs have Accessibility Issues - [#9078](https://github.com/NuGet/Home/issues/9078)
+
+* FireAndForget PM UI startup - [#9112](https://github.com/NuGet/Home/issues/9112)
+
+* Accessibility Fixes in PM UI - [#9157](https://github.com/NuGet/Home/issues/9157)
+
+* Restore: IncludeExcludeFiles.Equals(...) implementation is incorrect - [#9167](https://github.com/NuGet/Home/issues/9167)
+
+* Restore: PackageSpec.Clone() creates unequal clone - [#9211](https://github.com/NuGet/Home/issues/9211)
+
+* Error list shown although "Always show Error List if build finishes with errors" is not checked - [#8190](https://github.com/NuGet/Home/issues/8190)
+
+**None:**
+
+* Add more settings tests - completely cover project\sln level support across products - [#8927](https://github.com/NuGet/Home/issues/8927)
```

Comment:
I would just try to classify the nones. 
You can work with me or @zkat on this.